### DISCUSSION TITLE
Android Leela's speed is too slow.

### DISCUSSION THREAD

**Discussion by Frosty54** [LeelaChessZero/lc0#1866] [Q&A]
For Android Leela is possible to speed up kns?

**Comment by mooskagh** [comment]
Although the code for Android is not optimized as much as for desktop GPUs, it's likely that the performance is close to what device's CPU is capable of doing.

For low performance devices, it's usually advised to take one of the smaller networks from here: https://lczero.org/dev/wiki/best-nets-for-lc0/
